Firebase/Core/FIRLogger.m

```diff
@@ -37,6 +37,7 @@
 FIRLoggerService kFIRLoggerInstanceID = @"[Firebase/InstanceID]";
 FIRLoggerService kFIRLoggerInvites = @"[Firebase/Invites]";
 FIRLoggerService kFIRLoggerMessaging = @"[Firebase/Messaging]";
```

Comment:
nit: flip to match ordering in FIRLogger.h?

Reply:
Whoops, thanks! `clang-format` swapped it on me :)